get-env.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# Build a config/env file with informative key-values about the environment
+# Usage: ./get-enh.sh [context] [optional: file extension | default: yml] [optional: env file dir | default: $PWD]
+
+# Replaces/Creates a .env.<context>.<extension> in the $PWD or provided dir path
+
+exec 2> /dev/null
+
+if [ -z "$1" ]
+  then
+    echo "Please provide a CONTEXT argument"
+    exit 1
+fi
+
+CONTEXT=$1
+EXTENSION=${2:-yml} # default to YAML if no arg
```

Comment:
Not a problem but just making you aware / reminding that if we want to customize `ENV_FILE_DIR` then we need to pass `EXTENSION` as well in the CLI, since you're reading them as $2 and $3

Reply:
Yeah, i subconsciously put adding `case .. esac` or `getopts` with flags as a TODO if this script will be used in additional places, wrote TODO this time.